src/web/pages.rkt

```diff
@@ -1,32 +1,19 @@
 #lang racket
 
-(require (only-in fpbench fpcore? supported-by-lang? core->js js-header) json)
-(require "../alternative.rkt" "../syntax/read.rkt" "../sandbox.rkt" )
-(require "common.rkt" "timeline.rkt" "plot.rkt" "make-graph.rkt" "traceback.rkt"
-        "../syntax/sugar.rkt" "../float.rkt" "../syntax/types.rkt" "../syntax/syntax.rkt")
+(require json (only-in fpbench fpcore? supported-by-lang? core->js js-header))
+(require "../syntax/read.rkt" "../syntax/sugar.rkt" "../syntax/syntax.rkt" "../syntax/types.rkt"
+         "../alternative.rkt" "../float.rkt" "../points.rkt" "../sandbox.rkt"
+         "common.rkt" "timeline.rkt" "plot.rkt" "make-graph.rkt" "traceback.rkt")
 (provide all-pages make-page page-error-handler)
 
 (define (unique-values pts idx)
   (length (remove-duplicates (map (curryr list-ref idx) pts))))
 
 (define (all-pages result)
-  (define test (test-result-test result))
-  (define good? (test-success? result))
-  (define others
-    (if good?
-      (let ([other (cdr (test-success-end-alts result))]
-            [vars (test-vars test)])
-        (if (and good? (< (* (length other) (length vars)) 100))
-            (build-list (length other) (λ (x) (format "o~a" x)))
-            '()))
-      '()))
-
-  (define pages
-    `("graph.html"
-      ,(and good? "interactive.js")
-      "timeline.html" "timeline.json"
-      ,(and good? "points.json")))
-  (filter identity pages))
+  (define good? (eq? (test-result-status result) 'success))
+  (define default-pages '("graph.html" "timeline.html" "timeline.json"))
+  (define success-pages '("interactive.js" "points.json"))
+  (if good? (append default-pages success-pages) default-pages))
```

Comment:
I would write this a little differently, as `(append default-pages (if good? success-pages empty))`, but it's fine either way.